tests/network/vmi_infosource.go

```diff
@@ -133,6 +133,10 @@ var _ = SIGDescribe("Infosource", func() {
 				return dummyInterfaceExists(vmi)
 			}, 120*time.Second, 2*time.Second).Should(Equal(true))
 
+			networkInterface := netvmispec.LookupInterfaceStatusByMac(vmi.Status.Interfaces, primaryInterfaceMac)
```

Comment:
Please also verify the `ip` is not `nil` for the ga ones.

Reply:
one of them is a dummy that we are creating without ip.

the other (the one after the mac change) i can verify, but it might be flaky because it is not marked as unmanaged by network manager,
and after few seconds it will lose its ip IIRC (saw this behavior when was watching the status manually)

I think that there are enough tests that check the guest agent report,
unless you insist to include it here as well ?